UserOperation.Web/Controllers/StabilityController.cs

```diff
@@ -32,36 +42,33 @@ public IActionResult Index()
         }
         public IActionResult Create()
         { 
-            var projectList = _mapper.Map<List<ProjectViewModel>>(_stabilityHelper.GetProjects());
-            ViewBag.ProjectListDB = projectList;
-
-            var levelList = _mapper.Map<List<LevelViewModel>>(_stabilityHelper.GetLevels());
-            ViewBag.LevelListDB = levelList;
-
-            var stabilityLevelList = _mapper.Map<List<StabilityLevelViewModel>>(_stabilityHelper.GetStabilityLevels());
-            ViewBag.StabilityLevelListDB = stabilityLevelList;
-
-            var criticalityList = _mapper.Map<List<CriticalityViewModel>>(_stabilityHelper.GetCriticalities());
-            ViewBag.CriticalityListDB = criticalityList;
-
-            var positionList = _mapper.Map<List<PositionViewModel>>(_stabilityHelper.GetPositions());
-            ViewBag.PositionListDB = positionList;
+            ViewBag.ProjectListDB = _projects;    
```

Comment:
create a method in this controller, where you'll initialize the viewbag items, and just call it in create and edit